src/ui/common/src/components/layouts/menuSidebar.tsx

```diff
@@ -1,7 +1,5 @@
 import {
-  faBell,
```

Comment:
lots of lint fixes after I ran lint over common.

Not sure what's up here, but they should be fixed. 

Thought it was a node version issue, but I checked out all outstanding changes and re-ran nvm use from the ui directory before linting again and still got the same output.